apps/api/src/scripts/generate-config.ts

```
@@ -0,0 +1,575 @@
+`);
+}
+
+function convertValue(
+  value: string,
+  type: "string" | "number" | "boolean" | "array",
+): any {
+  switch (type) {
+    case "boolean":
+      return value.toLowerCase() === "true";
+    case "number":
+      const num = Number(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)